tensorflow/compiler/mlir/tosa/BUILD

```diff
@@ -79,6 +79,7 @@ cc_library(
         "//tensorflow/compiler/mlir/lite:tensorflow_lite",
         "//tensorflow/core:framework",
         "//tensorflow/core/kernels:conv_grad_shape_utils",
+        "//tensorflow/core/kernels:fake_quant_ops",
```

Comment:
It implements tensorflow::Nudge that is invoked from convertFakeQuantOp() in legalize_common.cc

Reply:
Thanks - i will try to review in detail tonight but have a lot in my queue and it may slip to tomorrow.

Reply:
This will add a cross dependency on a large body of TensorFlow to the TFLite path (est. 1000 additional source files?). I would rather just copy the Nudge function locally. (this has been done a couple of times for dependency reasons already)

Reply:
sure we can do that and remove the call to Nudge.

Reply:
Thanks for suggestion. Will try to replace it with a local implementation in next round.